webview/src/setup/components/studio/Studio.tsx

```diff
@@ -22,9 +27,32 @@ export const Studio: React.FC<{
     )
   }
 
+  const children = (
+    <DetailsTable testId="studio-url-details">
+      <DetailsTableRow
+        title="Self-Hosted Url"
+        text={selfHostedStudioUrl || 'Not found'}
+        actions={
+          selfHostedStudioUrl
+            ? [
+                {
+                  onClick: saveStudioUrl,
+                  text: 'Update'
+                },
+                {
+                  onClick: removeStudioUrl,
+                  text: 'Remove'
+                }
+              ]
+            : [{ onClick: saveStudioUrl, text: 'Add Url' }]
```

Comment:
[Q] Should this be "Add"? Does the text appear too short?

Reply:
> [Q] Should this be "Add"? Does the text appear too short?

Yes, the text appeared too short :)